articles/openshift/oshift-how-build-app-private-repo.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to build an OpenShift application from a private Github repo | UKCloud Ltd
+description: Shows how to build an OpenShift application where the source code is in a "private" github repository secured with an SSH Key Pair.
```

Comment:
private doesn't need to be in quotes
github = GitHub
Key Pair = key pair
No full stop needed at the end